client/tooltool.py

```diff
@@ -47,6 +48,8 @@
 from random import random
 from subprocess import PIPE, Popen
 
+import certifi
```

Comment:
The tooltool client currently has no non-stdlib dependencies (https://github.com/mozilla-releng/tooltool/blob/7b3324ef4da80fdf860eb0a947345f4f6fd65eb4/client/requirements/base.in), which seems like something we might want to preserve, especially since this issue is windows only, so we probably want to make this optional.
(And if not then we need to update the requirements files instead)

Reply:
I agree with Julien here, currently I believe there are places that just assume we can copy/paste this file around to wherever it's needed. If we need dependencies, then we should probably also look into making a proper package out of it.

Reply:
Sounds good, this was a simple change to get tooltool working for win11 source tests, I'll close this as it seems Joel has a working tooltool script in-tree.